linera-service/src/node_service.rs

```diff
@@ -960,20 +958,18 @@ where
         }
     }
 
-    #[expect(clippy::type_complexity)]
-    pub fn schema(&self) -> Schema<QueryRoot<P, S>, MutationRoot<P, S, C>, SubscriptionRoot<P, S>> {
+    pub fn schema(&self) -> Schema<QueryRoot<C>, MutationRoot<C>, SubscriptionRoot<C>> {
         Schema::build(
             QueryRoot {
-                clients: self.clients.clone(),
+                context: self.context.clone(),
```

Comment:
Nit: it's [recommended](https://doc.rust-lang.org/book/ch15-04-rc.html#using-rct-to-share-data#using-rct-to-share-data) to write this as `Arc::clone(&self.context)`.